pkg/apis/ceph.rook.io/v1/placement.go

```diff
@@ -90,6 +92,16 @@ func (p Placement) mergeNodeAffinity(nodeAffinity *v1.NodeAffinity) *v1.NodeAffi
 	return result
 }
 
+func (p Placement) mergeTolerations(tolerations []v1.Toleration) []v1.Toleration {
+	// no toleration is specified yet, return placement's toleration
+	if tolerations == nil {
+		return p.Tolerations
+	}
+
+	result := append(p.Tolerations, tolerations...)
```

Comment:
nit: no need for the `result` var
```suggestion
	return append(p.Tolerations, tolerations...)
```